src/test/java/it/pagopa/pn/downtime/DocumentCompositionTest.java

```diff
@@ -0,0 +1,78 @@
+package it.pagopa.pn.downtime;
+
+import com.openhtmltopdf.pdfboxout.PdfRendererBuilder;
+import org.junit.jupiter.api.Test;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class DocumentCompositionTest {
+
+    public class DocumentComposition {
```

Comment:
We don't need a class into another. We can move html2Pdf under DocumentCompositionTest and use it directly